script/generate-version.js

```diff
@@ -27,6 +23,16 @@ try {
   });
 }
 
+let description;
+try {
+  exec(`git fetch --tags https://github.com/aws/amazon-chime-sdk-js`);
```

Comment:
Move this to its own `try` block: the error message below will be confusing if fetching tags fails because _e.g._, the building machine is offline, and if this fetch fails you should still move on to try the next clause, because we might have the correct tags already.

Reply:
yup, thanks. Fixed